src/_locales/ru/messages.json

```diff
@@ -27,7 +27,7 @@
 		"message": "Фоновый скрипт не отвечает"
 	},
 	"message_translateOfText": {
-		"message": "Перевод"
+		"message": "Переведено"
```

Comment:
This key in not used. TODO: remove it

Reply:
["message": "Translated"](https://github.com/translate-tools/linguist/blob/dd62177b28214151a39caee3d64edfbe5b0be84d/src/_locales/en/messages.json#L30)

Reply:
It's not for you, it's note for me